crates/cli/src/package_import.rs

```diff
@@ -52,6 +56,74 @@ impl ImportMethodImpl for PackageImportMethod {
     }
 }
 
+/// This function does 2 things:
+/// 1. Install the files from `cas_paths`
+/// 2. Create the symlink layout
+///
+/// **TODO:** may break this function into 2 later
```

Comment:
In js pnpm we actually create all the symlinks in a separate process in parallel. The benefit of this approach is that we are not waiting for the packages to be fetched from the registry. We can already create all the symlinks while still downloading packages. That's fine because symlinks can point to directories that don't exist.

Your measurements show that this is already faster than pnpm, so we can keep it as it is for now.